### PR TITLE
Add Marching Orders, Clappy Trio and Rhythm Test stuff

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly
  * VIDEO COMING SOON!

**Issues Fixed:** [Any issues fixed by this pull request]

### Changes Proposed in this Pull Request
* Add 3 new games - Marching Orders, GBA Clappy Trio (Fever sounds moved to the Fever folder), and Rhythm Test.

### CODE REVIEW

**Review by chrislo27 — COMMENTED** [review]
Looks good from a quick read through - could you find a way to make the two Clappy Trio icons differ obviously?

File: core/assets/sounds/cues/marchingOrdersEn/data.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
+{
+  "gameID": "marchingOrdersEng",
```
Comment (chrislo27) [inline core/assets/sounds/cues/marchingOrdersEn/data.json]: Please refer to the [language guidelines](https://github.com/chrislo27/RhythmHeavenRemixEditor2/wiki/Guidelines-and-Syntactical-Sugar#language-guidelines). Use `marchingOrdersEn` (you did this in data.json which is right but not in here for some reason).

**Review by chrislo27 — CHANGES_REQUESTED** [review]

**Comment by GuardedLolz** [comment]
Done.

File: core/assets/sounds/cues/marchingOrdersEn/data.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
+{
+  "gameID": "marchingOrdersEn",
+  "gameName" : "Marching Orders (English)",
+  "series" : "tengoku",
+  "cues": [
+    {
+      "id": "marchingOrdersEng/attention1",
```
Comment (chrislo27) [inline core/assets/sounds/cues/marchingOrdersEn/data.json]: IDs are still on old Eng.

**Review by chrislo27 — CHANGES_REQUESTED** [review]

**Comment by chrislo27** [comment]
Also, the FEVER and GBA signs on Clappy Trio are impossible to read at a glance. It should be much more obvious.

**Comment by chrislo27** [comment]
I merged it because I am going to fix the errors by a certain time. There are several (missing Engs left over, icons not changed) so please next time fix all the errors.

**Pull request merged**